docs/walkthrough/create-training-data.md

```diff
@@ -41,6 +24,19 @@ I'm (…) sorry!          apologize-english
 I'm sorry to have…      apologize-english
 Please, forgive me!     apologize-english
 ```
+
+When using image-to-image retrieval models, images can be represented as a path to a file:
+
+```markdown
+green_apple.jpg                         picture of apple
+red_apple.jpg                           picture of apple
+https://example.com/apple-styling.jpg   picture of apple
+orange.jpg                              picture of orange
+https://example.com/orange-styling.jpg  picture of orange
+```
+
+If paths to local images are provided, they can be loaded into memory by setting `convert_to_blob = True` in the {class}`~finetuner.data.CSVOptions` object.
```

Comment:
can we do something like this, check if the uri is local or not, if so, apply load blob automatically. checkout the docarray implemnetation here: https://github.com/jina-ai/docarray/blob/main/docarray/document/mixins/helper.py#L15-L20

Reply:
yes we could, this could be added in pr #595